### DISCUSSION TITLE
Adding a row in q-table

### DISCUSSION THREAD

**Discussion by DaveDisco** [quasarframework/quasar#8868] [General - Components / Directives / etc]
I'm looking to add a row in q-table. Wherever I insert the new row in the data array I would need the table to scroll to the page that it appears on so that I can see it. Is there a way to do this? Or is there another approach I can take?

**Comment by ontwikkelfabriek** [comment]
@DaveDisco 

Here's a thread on the old forum about how to scroll to a row:
https://forum.quasar-framework.org/topic/5366/how-to-programmatically-scrollto-a-row-in-a-qtable/4
